pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -598,6 +598,43 @@ def write_traj(
             **kwargs
         )
 
+    def get_neighbors_snapshots(self, snapshot_indices=None, num_neighbors=12, **kwargs):
```

Comment:
Would it make sense to move this to the `Trajectory` class somehow?

Reply:
Ah, it's already on there anyway.  Then I'd want to think about whether this is really necessary to have it duplicated here.  It's just about typing

```python
job.trajectory().get_neighbors()
```
vs.
```python
job.get_neighbors()
```
?

Reply:
> 
> 
> Ah, it's already on there anyway. Then I'd want to think about whether this is really necessary to have it duplicated here. It's just about typing
> 
> ```python
> job.trajectory().get_neighbors()
> ```
> 
> vs.
> 
> ```python
> job.get_neighbors()
> ```
> 
> ?

Yes, I prefer the second one over the first mainly because it's available via auto-complete and the first isn't. Plus it is naturally equivalent to `structure.get_neighbors()`

Reply:
auto-complete is a fair argument.